optuna/visualization/_contour.py

```diff
@@ -302,7 +302,7 @@ def _generate_contour_subplot(
     # If contours_coloring='heatmap' is specified, reversescale argument of go.Contour does not
     # work correctly. See https://github.com/pfnet/optuna/issues/606.
     colorscale = plotly.colors.PLOTLY_SCALES["Blues"]
-    if direction == StudyDirection.MAXIMIZE:
+    if direction == StudyDirection.MINIMIZE:
```

Comment:
Can we omit this unrelated change from this PR?

Reply:
Sorry about that.